inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -684,10 +689,20 @@ public void onRecommendationRejectEvent(AjaxRecommendationRejectedEvent aEvent)
                 return;
             }
 
-            if (document.equals(annotatorState.getDocument())
-                    && vid.getLayerId() == selectedLayer.getObject().getId()
-                    && prediction.get().equals(currentRecommendation)) {
-                
+            AnnotationObject rejectedRecommendation = prediction.get();
+            applicationEventPublisherHolder.get().publishEvent(
+                new ActiveLearningRecommendationEvent(this, eventState.getDocument(),
+                    rejectedRecommendation, annotatorState.getUser().getUsername(),
+                    selectedLayer.getObject(), rejectedRecommendation.getFeature(),
+                    LearningRecordUserAction.REJECTED.toString(), model
```

Comment:
Can you pass a LearningRecordUserAction enum value instead of its string value, or would that cause dependency problems?